src/passwordless/passwordless.spec.ts

```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+import MockAdapater from 'axios-mock-adapter';
+
+import { WorkOS } from '../workos';
+
+const mock = new MockAdapater(axios);
+
+describe('Passwordless', () => {
+  afterEach(() => mock.resetHistory());
+
+  describe('createSession', () => {
+    describe('with valid options', () => {
+      it('creates a passwordless session', async () => {
+        mock.onPost().reply(200, {});
```

Comment:
Does this return a 200 or a 201?

Reply:
Recently I started using a fixture pattern of responses so we not only test that the SDK forms the proper request but we also test that it handles the response properly. You can see an example here. Might be nice to throw in!

https://github.com/workos-inc/workos-node/pull/218/files#diff-23bdba8287c2e5e301c52a9d04fe0c46R19

Reply:
Just checked and it is a 201, I can update it. And thanks for the share, I'll update the test to follow that pattern.

Reply:
- [ ] update test to use fixtures
- [ ] update test response to be a 201